rpa-manager/modules/ROOT/pages/botmanagement-support.adoc

```diff
@@ -54,27 +54,39 @@ Copy an API key to the clipboard to paste it into the according field during ins
 
 == Manage Service Times
 
-Service times are any repeatable time intervals that define when a service is available. You assign them to bots to determine when a bot may execute processes.
+Service times are repeatable time intervals that define when a service is available. You assign them to bots to determine when a bot may execute processes.
 
 You can use a service time to simulate the regular weekly working hours at your company, for example. The bot can then work at the same time as its human colleagues if tasks are interdependent, or at different times if you want it to use resources that other employees are not currently using.
 
-Processes that are deployed to the bot are only executed by the bot if it is within one of its service times.
+Bots execute processes deployed to them only if they are within one of their service times.
 
-You can assign multiple service times to a bot. If you do not assign any service times to a bot, it may work 24/7.
+You can assign multiple service times to a bot. If you do not assign any service times to a bot, it works 24/7.
 
-A service time is specified in a specific time zone. If the bot is in a different time zone, the time interval is converted accordingly.
+You specify the service time in one time zone. If the bot that you assign the service time to is installed on a computer using a different time zone, the service time interval is converted to the bot’s time zone. If you assign the service time to multiple bots in different time zones, all bots will execute processes simultaneously. The time intervals may change depending on daylight saving time.
+
+For example, if you specify a service time starting at 9:30 AM and ending at 5:30 PM in Buenos Aires time (ART (UTC -3)) and the bot is installed on a computer in Berlin (CET (UTC +1)), the bot will execute processes between 1:30 PM and 9:30 PM in Berlin time.
 
 You can create, edit, and remove service times.
 
+Learn how to xref:botmanagement-manage.adoc#bot-assign-servicetimes-applications[assign service times to a bot].
+
 == Create a Service Time
 
 Create service times to coordinate bot working hours.
 
-The names of the service times do not have to be unique. For example, you can assign the same name to a service time on working days for the business hours for different time zones. However, we recommend using different names, or at least different extensions to the same name, to rule out misunderstandings during interpretation of the effective schedule runtimes.
+The names of the service times do not have to be unique. To avoid misunderstandings of the interpretation of the effective schedule runtimes, include the time zone in the name of the service time.
```

Comment:
```suggestion
The names of the service times do not have to be unique. To avoid misunderstandings of the effective schedule runtimes, include the time zone in the name of the service time.
```